docs/explore_api/tasks.md

```diff
@@ -0,0 +1,210 @@
+>>> task1 = inductiva.tasks.Task("i4ir3kvv62odsfrhko4y8w2an")
+>>> task2 = inductiva.tasks.Task("i4ir3kvv62odsfrhko4y8w2an")
+>>> print(task1.id, task2.id)  # Outputs the same task ID
+```
+
+## Task Execution
+
+Tasks run **asynchronously** by default, allowing batch submissions in a single 
+session without waiting for each to complete. The API distributes them across 
+the available computational resources so that they can run in parallel. You 
+can manage a task's completion by either using the `wait` method or 
+the `sync_context` manager. 
+
+### `wait`
+
+You can turn a task into a blocking call by using the `wait` method.
+This method will block the call, allowing your script to wait until the task comes 
+to a terminal status without affecting the remote simulation if it were locally 
+interrupted (ex. the script is abruptly terminated). 
+
+This is useful when you want to wait for the completion of a task before proceeding 
+with some other operation (ex. downloading the output files):
+
+```python
+# Block the call until the simulation completes
+task.wait()  # <- The remote simulation WILL NOT DIE if the local session is
+             #    interrupted while waiting for the wait() call to return
+```
+
+### `sync_context` 
+
+Alternatively, you can use the `sync_context` manager to ensure the remote simulation 
+terminates if your local session is interrupted while waiting for the `wait` call 
+to return (e.g., through a Ctrl+C command):
+
+```python
+# Block the call until the simulation completes
+with task.sync_context():
+    task.wait()  # <- The remote simulation WILL DIE if the local session is
+                 #     interrupted while waiting for the wait() call to return
+```
+
+This setup provides flexibility in managing simulations, whether you prefer to 
+proceed with other tasks while simulations run or need to ensure completion before 
+moving forward. 
+
+## Task Lifecycle
+
+The status of a task changes as it progresses through its lifecycle, transitioning 
+through a myriad of intermediate states between creation to completion or termination.
+Thus, tasks are characterized by their state. Understanding these states is crucial 
+if you want to track a task's progress through the API and manage your simulations 
+effectively. 
+
+The following diagram shows the path a task may take through the API, and identifies 
+the relevant states and possible state transitions:
+
+
+<div align="center">
+   <img src="../_static/task_state.svg" alt="Task state diagram">
+   <figcaption align = "center"><b>State diagram of a Task</b></figcaption>
+
+</div>
+
+
+Below a succinct description of each state, including the actions that
+lead to a state transition:
+
+<details> <summary> 
+
+`PENDING INPUT`</summary>
+After you make your initial simulation request, the newly
+created task remains in this pending state, awaiting all necessary input 
+files. You can cancel the task, moving it to `KILLED`, or it may become `ZOMBIE` 
```

Comment:
not really cancel because there is no such concept.

```suggestion
files. You can kill the task, moving it to `KILLED`, or it may become `ZOMBIE` 
```